app/controllers/lookups_controller.rb

```diff
@@ -1,8 +1,11 @@
 class LookupsController < ApplicationController
+  DEFAULT_STATES_FILTER = %w(published unpublished)
```

Comment:
good point

On Tue, 12 Jul 2016 at 13:54 Kevin Dew <notifications@github.com> wrote:

> In app/controllers/lookups_controller.rb
> <https://github.com/alphagov/publishing-api/pull/410#discussion_r70431364>
> :
>
> > @@ -1,8 +1,11 @@
> >  class LookupsController < ApplicationController
> > +  DEFAULT_STATES_FILTER = %w(published unpublished)
>
> can we run freeze on this? consistent with other array constants in app
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/alphagov/publishing-api/pull/410/files/5e5ca247a908cebf44cac3ecac46d7016acf2d13..e06f39feb2b12111b4f0163262bc6698dbacb46a#r70431364>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAL3z2bAGFcy54TTzZSrSbQiUw4QhZAeks5qU45ngaJpZM4JJapM>
> .
>